tools/cldr-unittest/src/org/unicode/cldr/unittest/TestCLDRLocaleCoverage.java

```diff
@@ -171,7 +171,7 @@ static final StringBuilder composeList(Iterable<String> source, String separator
     }
 
     private boolean assertContains(String title, Collection<String> set, Collection<String> subset) {
-        boolean result = assertTrue(title, set.containsAll(subset));
+        boolean result = set.containsAll(subset);
```

Comment:
Thought i fixed this before. Basically, this makes sure we get some details about what locales were missing, not just 'assert failed'.

The issue here was that this PR had incorrectly added `lij` to language80 coverage rules. That's no longer in this PR, but this makes sure that the resulting err is clear.